translationCoreApps/tC_Resources
# WIP: toChapterFiles with tests

## This PR Addresses

Adding a helper to convert a usfm file from path to json chapter files given a path.

## To Test this PR

- Run `npm install` inside `tC_resources` path to install node_modules
- Run `jest __tests__/usfmToJsonHelpers.test.js` to verify that the new tests pass to ensure functionality.

## Known issues

- All other tests are not passing, that was pre-existing and needs to be addressed in another issue.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/tc_resources/11)
